src/apps/companies/apps/exports/tasks.js

```diff
@@ -38,14 +39,15 @@ const transformFullExportHistory = ({ count, results }) => ({
   results: results.map(createCountry),
 })
 
-const handleError = (e) => Promise.reject(Error(e.response.data.detail))
-
-export const fetchExportsHistory = ({ companyId, activePage }) =>
-  axios
-    .post('/api-proxy/v4/search/export-country-history', {
+export const fetchExportsHistory = apiTaskFactory(
+  'v4/search/export-country-history',
+  transformFullExportHistory,
+  ({ companyId, activePage }) => ({
+    method: 'post',
+    data: {
```

Comment:
I would be easier to understand (and less magical)  if `fetchExportsHistory` was a simple function.
```suggestion
export const fetchExportsHistory = ({ companyId, activePage }) => apiTaskFactory(
  'v4/search/export-country-history',
  {
    method: 'post',
    data: {
      company: companyId,
      limit: 10,
      offset: activePage * 10 - 10,
    },
  },
  transformFullExportHistory,
)
```